ang/civicase/CaseDetails.js

```diff
@@ -256,6 +242,34 @@
       return item;
     }
 
+    /**
+     * Prepare Related Cases
+     * @param {*} caseObj
+     */
+    function prepareRelatedCases (caseObj) {
+      caseObj.relatedCases = _.each(_.cloneDeep(caseObj['api.Case.getcaselist.relatedCasesByContact'].values), formatCase);
+      // Add linked cases
+      _.each(_.cloneDeep(caseObj['api.Case.getcaselist.linkedCases'].values), function (linkedCase) {
+        var existing = _.find(caseObj.relatedCases, {id: linkedCase.id});
+        if (existing) {
+          existing.is_linked = true;
+        } else {
+          linkedCase.is_linked = true;
+          caseObj.relatedCases.push(formatCase(linkedCase));
+        }
+      });
+
+      caseObj.relatedCases.sort(function (x, y) {
+        return !!y.is_linked - !!x.is_linked;
+      });
+
+      $scope.$emit('civicase::fetchMoreContactsInformation', caseObj.relatedCases);
```

Comment:
I know this was not introcuded by this PR, but this is a side effect: I requested to format a simple case object and all of a sudden a different part of the system is being called? Can you please check the purpose for this `fetchMoreContactsInformation` event?

Reply:
yes, so this is about fetching more information for the contacts in these related cases, which is used to in contact-card. As this contact fetching for cases, used in multiple places, a event has been used, which is caught by https://github.com/compucorp/uk.co.compucorp.civicase/blob/develop/ang/civicase/CaseListTable.js#L381.